style.css

```diff
@@ -3338,4 +3339,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```

Comment:
@ntwb Thanks for your suggestion! As I checked, at this moment - there is newline exist at the end of style.css

Could you please suggest if there is any workaround for me?